server/serverAPI/routes/AuthenticationRoutes.ts

```diff
@@ -25,7 +25,7 @@ const authenticationController = new AuthenticationController(
     new TokenCreator(privateKey)
 );
 
-authenticationRoute.use(express.urlencoded({ extended: true }));
+authenticationRoute.use(express.json());
```

Comment:
A day of work for one line change :cry:

Reply:
don't remind me about it.. still :angry: